tests/tracing/test_fluent.py

```diff
@@ -82,7 +82,62 @@ def square(self, t):
     }
 
 
-def test_trace_handle_exception_during_prediction(mock_client):
+def test_trace_in_databricks_runtime(clear_singleton, mock_store, monkeypatch):
+    monkeypatch.setenv("DATABRICKS_RUNTIME_VERSION", "1")
+    monkeypatch.setenv("MLFLOW_EXPERIMENT_NAME", "test")
+    mock_store.get_experiment_by_name().experiment_id = "test_experiment_id"
+
+    class TestModel:
+        @mlflow.trace()
+        def predict(self, x, y):
+            z = x + y
+            z = self.add_one(z)
+            z = mlflow.trace(self.square)(z)
+            return z  # noqa: RET504
+
+        @mlflow.trace(
+            span_type=SpanType.LLM, name="add_one_with_custom_name", attributes={"delta": 1}
+        )
+        def add_one(self, z):
+            return z + 1
+
+        def square(self, t):
+            res = t**2
+            time.sleep(0.1)
```

Comment:
Do we need this sleep?

Reply:
We don't need it now. Back then the `TraceInfo` was created from the real timestamp so it was to validate if the latency is measured correctly. But now we mock out backend so the timestamp is dummy one anyway.